src/Components/Helper/Helper.js

```diff
@@ -0,0 +1,15 @@
+const Actions = require('./Action/Action.json'); 
```

Comment:
The file name is not specific enough. The file name should include the scope of responsibility.
Ex: rename to `Todo.helper.js` or `TodoHelper.js`

- [ ] refactor